src/ui/popup.js

```diff
@@ -482,11 +489,18 @@ export default class Popup extends Evented {
      *
      * @example
      * let popup = new mapboxgl.Popup()
-     * popup.toggleClassName('toggleClass')
+     * const popup = new mapboxgl.Popup({className: 'highlighted'})
+        .setText('Popup')
+        .setLngLat([0, 0])
+        .addTo(map);
+     * popup.toggleClassName('highlighted')
+     * popup.toggleClassName('highlighted')
      */
     toggleClassName(className: string) {
         if (this._container) {
             return this._container.classList.toggle(className);
+        } else {
```

Comment:
Super nitpicky 🙈 : can omit `} else {` with early return.

```suggestion
        }
```